packages/storybook/src/nimble/text-field/text-field-matrix.stories.ts

```diff
@@ -132,6 +134,31 @@ const component = (
     </${textFieldTag}>
 `;
 
+// prettier-ignore
+const reducedStateComponent = (
```

Comment:
Maybe requiredVisibleStatesComponent?
Just the stuff interesting for the requiredVisible feature